.xo-config.json

```diff
@@ -47,7 +47,7 @@
         "svglint.config.mjs",
         "svgo.config.mjs"
       ],
-      "nodeVersion": ">=18"
+      "nodeVersion": ">=18.4"
```

Comment:
So `nodeVersion` could still be pinned at `>=18`.

Reply:
> The fs/promises.constants is still an experimental feature and is not supported until Node.js 18.4.0.

Reply:
Ah, I see.